core/pkg/resourcehandler/filesloader.go

```diff
@@ -131,7 +132,12 @@ func getResourcesFromPath(path string) (map[string]reporthandling.Source, []work
 
 		var lastCommit reporthandling.LastCommit
 		if gitRepo != nil {
-			commitInfo, _ := gitRepo.GetFileLastCommit(source)
+			commitInfo, err := gitRepo.GetFileLastCommit(source)
+			if err != nil && !warnIssued {
```

Comment:
I think here we should check the error type, because in the future we might have different reasons for failing

Reply:
@matthyx I agree. However, I've attempted to percolate changes gradually: this PR focuses only on supporting a working build without full git support. IMHO, the suggested change (which I agree with) should be deferred to another PR.

Reply:
fair enough... let's do it in a follow up